spring-cloud-function-context/src/main/java/org/springframework/cloud/function/context/catalog/BeanFactoryAwareFunctionRegistry.java

```diff
@@ -90,11 +90,10 @@
  *
  * @author Oleg Zhurakousky
  * @author Eric Botard
- *
  * @since 3.0
  */
 public class BeanFactoryAwareFunctionRegistry
-		implements FunctionRegistry, FunctionInspector, ApplicationContextAware {
+	implements FunctionRegistry, FunctionInspector, ApplicationContextAware {
```

Comment:
are there only formatting changes in this file or did I miss something?

Reply:
Very possible indeed. I had to fight with Checkstyle for a while. That second commit is actually just that

Reply:
No, there is an actual change which is the one I wanted to discuss later on

Reply:
Ah, sorry did not catch that. I guess the change in existing code is:
- wrap MCs with the new class
- Set the `accept` header instead of `contentType`